pkg/golinters/unused/unused.go

```diff
@@ -90,7 +90,7 @@ func getUnusedResults(pass *analysis.Pass, settings *config.UnusedSettings) unus
 	opts := unused.Options{
 		FieldWritesAreUses:     settings.FieldWritesAreUses,
 		PostStatementsAreReads: settings.PostStatementsAreReads,
-		ExportedIsUsed:         settings.ExportedIsUsed,
+		ExportedIsUsed:         true,
```

Comment:
I think you said befor that you want `golangci-lint` to have its own default but since this stems from a bug where we're deviating from upstream, are you considering using the value from the [DefaultConfig](https://github.com/dominikh/go-tools/blob/d0a0c6521c9caf7ab90e19205990ebba45cee788/unused/unused.go#L241-L249) in `unused`?

I know it probably doesn't make any difference now but it serves some kind of documentation purpose and will follow any changes made upstream.

Reply:
In this context, we are forced to override the value to be sure that this value is always `true`.
I will not use `unused.DefaultOptions.ExportedIsUsed` because it can be mutated by `unused` itself (because it's a global variable), or the default value can be changed by `unused` (this is a problem for us).
I added a comment to reference the issue.